### PR TITLE
ticlient: import ticlient

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,7 +49,7 @@ godep:
 	go get -d github.com/pingcap/go-themis
 	go get -d github.com/pingcap/tso/client
 	go get -d github.com/pingcap/pd/pd-client
-	go get -d github.com/pingcap/ticlient
+	go get -d gopkg.in/fatih/pool.v2
```
Comment (qiuyesuifeng) [inline Makefile]: Seems ticlient has more dependency packages？

**Comment by shenli** [comment]
Do we need to put ticlient into store package? Like store/hbase

File: Makefile (hunk 1 above)
Comment (coocood) [inline Makefile]: We can move those dependencies to 'vendor' later.

**Comment by coocood** [comment]
@shenli 
That's better, but have a package name `ticlient` in `store` lookup weird.

**Comment by ngaut** [comment]
Maybe we should rename ticlient to tikv.

**Comment by coocood** [comment]
@ngaut 
Agree.

**Comment by shenli** [comment]
LGTM
@disksing

**Comment by ngaut** [comment]
LGTM, but let @disksing to take a look.

**Comment by disksing** [comment]
LGTM.

**Pull request merged**